contracts/test/HardCodedConstantsTest.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { TokenManagerLiquidityPool } from '../token-manager/implementations/TokenManagerLiquidityPool.sol';
+import { Distributable } from '../utils/Distributable.sol';
+import { FlowLimit } from '../utils/FlowLimit.sol';
+import { NoReEntrancy } from '../utils/NoReEntrancy.sol';
+import { Operatable } from '../utils/Operatable.sol';
+import { Pausable } from '../utils/Pausable.sol';
+
+contract TestTokenManager is TokenManagerLiquidityPool {
+    string public constant NAME = 'TestTokenManager';
+
+    constructor(address interchainTokenService_) TokenManagerLiquidityPool(interchainTokenService_) {
+        require(TOKEN_ADDRESS_SLOT == uint256(keccak256('token-address')) - 1, 'invalid constant');
+        require(LIQUIDITY_POOL_SLOT == uint256(keccak256('liquidity-pool-slot')) - 1, 'invalid constant');
+    }
+}
+
+contract TestDistributable is Distributable {
```

Comment:
where are these being deployed in unit tests to call it?

Reply:
it seems like they got lost in all the rebasing, good catch. I re-added all the hardcoded tests for utils contracts, there should be 6 in total across `TokenManager.js` and `UtilsTest.js`